src/main/java/com/minecolonies/coremod/colony/requestsystem/management/IStandardRequestManager.java

```diff
@@ -3,8 +3,10 @@
 import com.google.common.collect.BiMap;
 import com.google.common.collect.ImmutableCollection;
 import com.google.common.reflect.TypeToken;
+import com.minecolonies.api.colony.requestsystem.data.*;
 import com.minecolonies.api.colony.requestsystem.manager.IRequestManager;
 import com.minecolonies.api.colony.requestsystem.request.IRequest;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.colony.requestsystem.request.IRequest'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)